src/components/PathwaysList/PathwayModal.tsx

```diff
@@ -49,6 +52,30 @@ const NewPathwayModal: FC<NewPathwayModalProps> = ({ open, onClose }) => {
     [addPathway, closeModal]
   );
 
+  const handleUpdatePathway = useCallback(
+    (event: FormEvent<HTMLFormElement>): void => {
+      event.preventDefault();
+      if (editPath) {
+        if (pathwayNameRef.current?.value) editPath.name = pathwayNameRef.current?.value;
+        if (pathwayDescRef.current?.value !== '')
+          editPath.description = pathwayDescRef.current?.value;
+        const pathwayId = editPath.id;
+        const pathwayIndex = pathways.findIndex(pathway => pathway.id === pathwayId);
+        updatePathwayAtIndex(editPath, pathwayIndex);
```

Comment:
I know this isn't something new in this PR but I'm thinking perhaps we should update this function to take in an ID instead of an index. We never have the index naturally so we're constantly copying and pasting this `findIndex` line. It would be cleaner to have the update function handle that for us.

Reply:
Okay! I can either go through and fix it now or maybe we should create a new item for it?

Reply:
I think we can create a new task for it - just wanted to see if other people agreed with me first

Reply:
Whoops already dealt with it :(